pyupgrade.py

```diff
@@ -2229,7 +2231,7 @@ def _to_fstring(src: str, call: ast.Call) -> str:
     parts = []
     i = 0
     for s, name, spec, conv in parse_format('f' + src):
-        if name is not None:
+        if name is not None and not s.endswith(r'\N'):  # skip unicodedata.lookup
```

Comment:
this might have the same problem as above, not sure though 🤔

Reply:
It doesn't, since `r'"{} \\N{snowman}".format(a)'` is an invalid format string.

I've added it to the noop tests though.

Reply:
what about `r'{foo} \N{snowman}'.format(foo=a, snowman=b)`